src/main/resources/db/migration/V1_83__prod_issue_fix_amend_multiple_true_sessions.sql

```diff
@@ -0,0 +1,18 @@
+/*
+    Same as v1_80, but used to target specific referrals that were not correctly migrated.
+*/
+with action_plans_ranked_by_approved_at_desc as (
+    select id,
+           referral_id,
+           approved_at,
+           row_number() over (partition by referral_id order by approved_at desc) rank
+    from action_plan ap
+    where approved_at is not null
+    and referral_id in ('d7c7d7c1-df71-4501-9a0c-6649bcf31674', '4a95bf60-f5ab-4470-8988-addd705f0d5c')
+)
```

Comment:
Since we're only doing this for 4 action plans, why not just do simpler SQL directly on action_plan_session table?

I.e. 
```
update action_plan_session set latest_approved = false where deprecated_action_plan_id in ('','');
update action_plan_session set latest_approved = true where deprecated_action_plan_id in ('','');
```